rosidl_typesupport_opendds_c/CMakeLists.txt

```diff
@@ -39,9 +39,8 @@ ament_export_include_directories(include)
 
 ament_python_install_package(${PROJECT_NAME})
 
-add_library(${PROJECT_NAME} SHARED
-  src/identifier.cpp
-  src/wstring_conversion.cpp)
+add_library(${PROJECT_NAME} SHARED)
+OPENDDS_TARGET_SOURCES(${PROJECT_NAME} src/identifier.cpp src/wstring_conversion.cpp)
```

Comment:
Doesn't look this these have anything to do with OpenDDS.  I guess it doesn't hurt to use the CMake macro anyway, but it may be confusing.